notebooks/source/truncated_distributions.ipynb

```diff
@@ -0,0 +1,1964 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #5.        x, y = jnp.unique(samples, return_counts=True)
</pre><p>Maybe using <code>np.unique</code>?  I'm not sure if <code>ax.bar</code>  works with device arrays in the latest jax version.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pyro-ppl/numpyro/pull/1272/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='55'/>

Reply:
Ah, good point @fehiepsi thank you! Should be fixed now :)